corehq/apps/app_manager/static/app_manager/js/commcaresettings.js

```diff
@@ -219,7 +219,11 @@ hqDefine('app_manager/js/commcaresettings.js', function () {
             setting.valueToSave = ko.computed({
                 read: function () {
                     if (setting.enabled()) {
-                        return setting.value() || setting.computeDefault();
+                        if (setting.computeDefault()) {
+                            return setting.value() || setting.computeDefault();
+                        } else {
+                            return setting.value();
```

Comment:
can you confirm that `setting.value()` still returns `null` in the scenarios where the setting doesn't show up on the page or is not selected from a multiselect (or that if it returns an empty string it's fine?).

otherwise looks good. I'm not too familiar with this part of app manager so might ask @orangejenny @snopoke or @dannyroberts to take a quick look and confirm this won't break anything.